ERROR - 2024-07-22 00:15:08 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 00:15:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 00:15:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 00:15:08 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 00:31:39 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 00:31:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 00:31:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 00:31:39 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 01:54:29 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "front/login.php"
[Method: GET, Route: login]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('front/login.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('front/login', [], true)
 3 APPPATH\Controllers\Home.php(48): view('front/login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 01:55:52 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 01:55:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 01:55:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'silvina@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'silvina@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 01:55:52 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'silvina@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'silvina@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'silvina@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 02:00:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 02:00:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 02:01:03 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 02:01:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 02:01:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 02:01:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 02:02:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.c...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.c...', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 02:02:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('silvina@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'silvina@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 02:24:18 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 02:24:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 02:24:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 02:24:18 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 03:09:17 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 03:09:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 03:09:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 03:09:17 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 03:39:26 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 03:39:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 03:39:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 03:39:26 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 04:05:12 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'your_username'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'your_username', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 04:05:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-22 04:26:53 --> mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-22 04:26:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 04:26:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-22 04:26:53 --> [Caused by] mysqli_sql_exception: Table 'machado_m.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'manuel.machado.j.m@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-07-22 04:32:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-22 04:32:51 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'manuel_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 04:32:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-07-22 04:39:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-22 04:39:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 04:39:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(28): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-07-22 06:46:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-22 06:46:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-22 06:46:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-22 06:47:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 06:47:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar_form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-07-22 06:52:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-22 06:53:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'machado_m', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 06:53:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar_form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-07-22 06:56:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-22 06:56:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'php', 3306, '', 0)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#7 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\primer_proyecto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\primer_proyecto\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\primer_proyecto\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\primer_proyecto\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado....', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\primer_proyecto\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado....', Array, Array, 'email')
#6 C:\xampp\htdocs\primer_proyecto\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\primer_proyecto\app\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#9 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#10 C:\xampp\htdocs\primer_proyecto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\primer_proyecto\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\primer_proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\primer_proyecto\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-22 06:56:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: POST, Route: enviar_form]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('manuel.machado.j.m@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'manuel.machado.j.m@gmail.com', [...], [...], 'email')
 6 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\usuario_controller.php(22): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
